Add wildcard support for banned-modules (#319)
adamchainz/flake8-tidy-imports: README.rst

    [flake8]
    banned-modules = decimal.Decimal = Use ints and floats only.

> Entries ending in ``.*`` are treated as wildcards; the example will ban both ``import foo`` and ``import foo.bar``:

Reviewer: A little wordsmithing - unnecessary semicolon
```suggestion
Entries ending in ``.*`` are treated as wildcards. For example, to ban both ``import foo`` and ``import foo.bar``:
```

Author: Out of date, resolving
